frontend/src/constants.js

```diff
@@ -48,6 +48,16 @@ export const stateChoices = [
   { value: "WB", label: "West Bengal" }
 ];
 
+export const occupationChoices = [
+  {value: "Student - College/University(above 18 years old)", label: "Student - College/University(above 18 years old)"},
```

Comment:
I think we can remove the "above 18 years old" part. Even the College/University doesn't make much sense, but maybe leave it in, for now.